doc/reduce-traffic.md

```diff
@@ -28,7 +28,9 @@ calculating the target.
 
 Disabling listening will result in fewer nodes connected (remember the maximum of 8
 outbound peers). Fewer nodes will result in less traffic usage as you are relaying
-blocks and transactions to fewer nodes.
+blocks and transactions to fewer nodes. This would also prevent reachable nodes from
+periodically querying DNS servers when it is not needed (feature, which gives more
+privacy to the non-reachable nodes located in the same subnet).
```

Comment:
I think this in an implementation detail which people that want to reduce their traffic are not interested in learning about.

A release notes snippet would be a better place to put this, no?

Reply:
Yeah, I was hesitant about this part. Committed this for now simply to not ignore this. Unless someone expresses support for keeping this, I would remove this commit later.